misc/memory-connection-limits/src/error.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 Protocol Labs.
```

Comment:
Do we need this module or can it just live in `lib.rs`?

Reply:
Moved to `lib.rs`